serving/docker/fastertransformer.Dockerfile

```diff
@@ -80,8 +80,8 @@ ENV PATH=/usr/local/mpi/bin:${PATH} LD_LIBRARY_PATH=/usr/local/mpi/lib:${LD_LIBR
 RUN mkdir -p /opt/tritonserver/backends/fastertransformer && mkdir -p /opt/tritonserver/lib && \
     curl -o /opt/tritonserver/lib/libtritonserver.so https://publish.djl.ai/tritonserver/${triton_version}/libtritonserver.so && \
     curl -o /opt/tritonserver/backends/fastertransformer/libth_transformer.so https://publish.djl.ai/fastertransformer/${ft_version}/libth_transformer.so && \
-    curl -o /opt/tritonserver/backends/fastertransformer/libtransformer-shared.so https://publish.djl.ai/fastertransformer/${ft_version}/libtransformer-shared.so && \
-    curl -o /opt/tritonserver/backends/fastertransformer/libtriton_fastertransformer.so https://publish.djl.ai/fastertransformer/${ft_version}/libtriton_fastertransformer.so && \
+    curl -o /opt/tritonserver/backends/fastertransformer/libtransformer-shared.so https://publish.djl.ai/fastertransformer/llama/libtransformer-shared.so && \
```

Comment:
Should we pass {ft_version} instead of hard code llama?

Reply:
right now {ft_version} is set to `main` and for this change , we are not using FT main. Or are you suggesting to set `ft_version` to `llama` or `custom`?